volume/volume.go

```diff
@@ -321,3 +321,22 @@ func errInvalidMode(mode string) error {
 func errInvalidSpec(spec string) error {
 	return fmt.Errorf("invalid volume specification: '%s'", spec)
 }
+
+// ValidateTmpfsMountDestination validates the destination of tmpfs mount.
+// Currently, we have only two obvious rule for validation:
+//  - path must be absolute
+//  - path must not be "/"
+// We should add more rules carefully (#30166)
+func ValidateTmpfsMountDestination(dest string) error {
```

Comment:
Should we move this to `volume_unix`?
Since:

1. Windows does not support tmpfs
2. Even if it did, this is not the correct way to validate paths in Windows.